### PR TITLE
ActionHero unlink method

### PR DESCRIPTION
**Pull request by S3bb1**
fixes https://github.com/evantahler/actionhero/issues/849

### CODE REVIEW

File: site/source/includes/docs/core/plugins.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Plugins are loaded after all local actionhero project files, but initializers fo
 
 `api.config.general.paths.plugin` (loaded from `/config/api.js`) is an array which contains the search path for your plugins.  This will default to `./node_modules`, but you can add a local path to your project.  Once you have the plugin search paths set up, you use `npm run actionhero link -- --name nameOfPlugin` (or `./node_modules/.bin/actionhero link --name nameOfPlugin`, which is equivalent) to create links in your top-level project to the plugin.  This will also copy over any config files from the plugin into your project so you can modify them.  The act of "linking" simply creates a `myPlugin.link` file in each component of your top-level project (actions, tasks, etc) which tells actionhero to load up files at boot from that plugin.  
 
-You can delete all (or some) links at any time.
+You can delete the linked files with the "unlink" method using  `npm run actionhero unlink --name nameOfPlugin`.
```
Comment (evantahler) [inline site/source/includes/docs/core/plugins.md]: > ... If your plugin was installed via NPM, also be sure to remove it from your `package.json`

File: bin/templates/help.txt [hunk 2]
```diff
@@ -78,6 +79,10 @@ Descriptions:
     top-level project normally, you will have first installed the plugin
     via `npm install myPlugin`
 
+* actionhero unlink --name=[pluginName]
+  will remove the linked actions, tasks, initializers, etc from a plugin in your
+    top-level project. Please remove the config files manually
```
Comment (evantahler) [inline bin/templates/help.txt]: > ... If your plugin was installed via NPM, also be sure to remove it from your package.json

File: bin/methods/unlink.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  });
+
+  if(!pluginRoot){
+    binary.log('plugin `' + binary.argv.name + '` not found in plugin paths', 'warning', binary.config.general.paths.plugin);
+    return next(true);
+  }
+
+  var pluginRootRelative = pluginRoot.replace(binary.argv.linkRelativeBase, '');
+  binary.log('unlinking the plugin found at ' + pluginRootRelative);
```
Comment (evantahler) [inline bin/methods/unlink.js]: Add:
```js
binary.log('remember that config files will need to be deleted manually');
binary.log('If your plugin was installed via NPM, also be sure to remove it from your package.json');
```

**Comment by evantahler** [comment]
Looks great! and I agree with your decision to not delete config files.  I suggest that you add a few more notes, and then this looks great!

**Comment by S3bb1** [comment]
Yeah, removing config files would be more difficult to detect ... so the best way ist to delete them manually

**Comment by evantahler** [comment]
💥

**Pull request merged**